### ISSUE TITLE
Numeric getData does not return proper value for web2print

### ISSUE THREAD

**Issue by Cruiser13** [pimcore/pimcore#1639]
# Bug Report
### Expected behavior 
Same (correct) output in preview mode and in web2print pdf.
### Actual behavior  
getData() does return different values
#### Code example 
```php
if($this->editmode){
echo $this->numeric("amount1");
}
else{
if($this->numeric("amount1")->getData() === 0){
echo "You did input a zero";
}
}
```

Put a 0 in the number field while editing the document. In preview, you'll get the text "You did input a zero" but it'll not be printed on the pdf.

Preview mode will output a 0 as int(0) while web2print pdf will list a string(1) "0".
Probably connected to https://github.com/pimcore/pimcore/issues/678

**Comment by Cruiser13** [comment]
As a follow-up: The isEmpty() function of the numeric editable returns true for no input and a 0 inside the input. If you use the numeric for currency amounts for example, you can not decide wether you should display a 0,00 $ in the frontend or nothing because the info is missing.

**Comment by Cruiser13** [comment]
I wonder if numeric editable is designed for 0 values at all. If a numeric editable is being used inside a block, the 0s are deleted leaving an empty field on reload.

**Comment by maff** [comment]
Empty and zero are now properly handled (see 49ed7bd00853f5e3a3f3b9d888d61107b41d07cb).

**Comment by Cruiser13** [comment]
@maff this does not solve the issue that 0s dissapear if the document does reload.

**Comment by maff** [comment]
@Cruiser13 should be fixed with 4ecc86ab20dc8516f061190620ab4fa328e6b08b

**Comment by Cruiser13** [comment]
@maff thank you that fixed the issue for me.
